src/EnergyPlus/OutputProcessor.cc

```diff
@@ -6468,26 +6466,37 @@ void UpdateMeterReporting(EnergyPlusData &state)
     }
 
     // Helper lambda to locate a meter index from its name. Returns a negative value if not found
-    auto findMeterIndexFromMeterName = [&state](std::string const &name) -> int {
-        auto &op(state.dataOutputProcessor);
+    auto setupMeterFromMeterName =
```

Comment:
So renaming the lambda to express that it is also setting up the meter, makes sense.